crmsh/bootstrap.py

```diff
@@ -104,6 +105,179 @@ def init_qdevice(self):
                 cmds=self.qdevice_heuristics,
                 mode=self.qdevice_heuristics_mode)
 
+    def init_sbd_manager(self):
+        self.sbd_manager = SBDManager(self.sbd_device, self.diskless_sbd)
+
+
+class SBDManager(object):
+    """
+    Class SBDManager is for sbd configuration and sbd service management
+    """
+    sbd_config_dict = {
+            "SBD_PACEMAKER": "yes",
+            "SBD_STARTMODE": "always",
+            "SBD_DELAY_START": "no"
+            }
+
+    def __init__(self, sbd_device=None, diskless_sbd=False):
```

Comment:
Maybe better to call `sbd_devices`, as the value can be a list

Reply:
Changed